src/test/scala/com/databricks/labs/deltaoms/common/OMSCommonSuite.scala

```diff
@@ -124,6 +124,63 @@ class OMSCommonSuite extends QueryTest with SharedSparkSession with DeltaTestSha
     assert(OMSCommandLineParser.consolidateAndValidateOMSConfig(argsEmpty, omsConfig) == omsConfig)
   }
 
+  test("Spark Optional Configurations checks") {
+    spark.conf.set(OMSSparkConf.RAW_ACTION_TABLE, "test.rawactions")
```

Comment:
this could be moved to the test setup hook/fixture as well.

Reply:
Modified code to utilize inbuilt withSQLConf Spark test function.